flowise-embed/src/utils/index.ts

```diff
@@ -6,6 +6,8 @@ export const isEmpty = (value: string | undefined | null): value is undefined =>
 
 export const isNotEmpty = (value: string | undefined | null): value is string => value !== undefined && value !== null && value !== '';
 
+const fourMinutesInMilliseconds = 240000;
```

Comment:
Seria melhor utilizar o cálculo ao invés do valor absoluto, por questões de legibilidade
```suggestion
const fourMinutesInMilliseconds = 4 * 60 * 1000;